src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/SscsBulkScanExceptionRecordCallback.java

```diff
@@ -469,6 +571,28 @@ private ExceptionRecord autoExceptionCaseData(Map<String, Object> caseData, Stri
             .build();
     }
 
+    //FIXME: delete after bulk scan auto case creation is switch on
+    @SuppressWarnings("unchecked")
```

Comment:
Is this not just a repeat of the `exceptionCaseData` method? Maybe you can just add a parameter to that method for the form type if everything else is identical?

Reply:
We have one for SSCS1 form, auto exception case. Just for readability and may be further test cases, I kept sscs2 separate.